docs/hardware/VIVE/focus3_xre.mdx

```diff
@@ -0,0 +1,123 @@
+import ReactPlayer from 'react-player'
+
+# Vive Focus 3 / Vive XR Elite
+
+## Introduction
+
+The Vive Focus 3 and XR Elite are standalone VR headsets powered by the Qualcomm XR2, similar to the Quest 2 and Quest Pro headsets from Meta.
+Although by default having neither eye nor face tracking, the Focus 3 has 2 add-on modules that can be installed to add eye and face tracking capabilities,
+and the XRE has a single, combo module that can be installed to add both eye and face tracking capabilities.
+There is currently no way for VRCFT to simultaneously extract data from the headset and send eye and face tracking to the Vive standalone version of VRChat, so the following solutions are only for PCVR.
+
+Since September 2023, the Vive PCVR VR streamer programs (Vive Business Streaming and Vive Streamer Hub) have had the ability to control VRCFT avatars in VRChat on their own (by copying the VRCFT program's functions).
+While users can choose to simply forgo using VRCFT, we would recommend *against* using the Vive Streamer built-in outputs and following this guide to use VRCFT instead. 
```

Comment:
For the sake of appearing unbiased, could we have a list of reasons somewhere as to why the vive streamer is sub-par and how the compability with all avatars is the same